setup.py

```diff
@@ -26,6 +26,7 @@
     'recommonmark>=0.4.0',
     'Sphinx>=1.3.5',
     'sphinxcontrib-napoleon>=0.4.4',
+    'sphinx-rtd-theme>=0.1.9',
```

Comment:
One minor thing: could you also update [`bigchaindb/docs/requirements.txt`](https://github.com/bigchaindb/bigchaindb/blob/develop/docs/requirements.txt) to be in sync with the list of docs-related packages in setup.py?